docs(hub): document the Catalog (getting started + usage) (#9048)
camunda/camunda-docs: docs/components/hub/organization/manage-catalog/catalog-usage.md

---
id: catalog-usage
title: Use Catalog assets in Hub
sidebar_label: Use Catalog assets
description: "Discover and apply published Catalog assets in the Hub Catalog and Canvas, and manage assets as a Center of Excellence."
---

> Once your Center of Excellence (CoE) has [synced element templates to the Catalog](/components/hub/organization/manage-catalog/getting-started.md), delivery teams discover and apply them while modeling, and the CoE manages their lifecycle.

Reviewer: Since the target user for this guide is the delivery team, we should move it from the Organization level to the Workspace level documentation. Consider here: Workspace > Model business processes > Model

Author: Agreed 👍

## Discover assets in the Catalog

> Delivery team members browse the Catalog in Hub to find approved, ready-to-use element templates. For each asset, the Catalog shows its name, short description, category, tags, latest version, and icon.

Reviewer: I don't see how to supply an icon in the other guide.

Author: It is reused from the element template json. We'll make it clearer in the guide

You can narrow the list to find the right asset:

- **Free-text search** matches against the asset name, short description, category name, and tag names (case-insensitive).
- **Filter by category** to scope the list to a single area (for example, _Connectors_ or _Human Tasks_).
- **Filter by tags** to combine related assets across categories.
- **Sort** by name, creation date, or last updated date.

> Assets that were recently added or updated are highlighted so teams can spot new templates at a glance. Only published assets appear in the Catalog — unpublished assets are never shown to delivery teams.

Reviewer: ```suggestion
Assets that were recently added or updated are highlighted so teams can spot new templates at a glance. Only published assets appear in the Catalog — unpublished assets are hidden from delivery teams.
```

"never" doesn't feel right, considering an asset is only unpublished _after_ it was once already published and, therefore, visible. In fact, unpublished assets may actually be actively used if they were applied before unpublication.

## Apply assets in Canvas

To use a Catalog asset while modeling:

> 1. Open a BPMN diagram in the Hub modeler (Canvas).

Reviewer: Is "Canvas" the official name for "Hub modeler"? I think we should pick a name and use it in an official capacity rather than using both.

Author: I think Hub modeler is the term used in the docs, and canvas the internal team name. Let's call it Hub modeler

2. Select an element and apply a published element template from the Catalog.
3. Configure the element through the properties panel as defined by the template.

When the CoE publishes a newer version of a template you already use, Hub offers an **update** path on elements that reference the older version, so you can move to the latest version on your terms.

> ## Manage the Catalog as a Center of Excellence

Reviewer: I think we should remove this, as it's already covered in the other guide, and this guide targets the delivery team.

Author: Agreed 👍
